### PR TITLE
ETCD-574: Update TLS artifact descriptions according to registry requ…

### PR DESCRIPTION
**Pull request by tjungblu**
…irements

### CODE REVIEW

**Comment by tjungblu** [comment]
fyi @vrutkovs - in case your node name adjuster for the registry needs adjusting

**Comment by vrutkovs** [comment]
LGTM for me, thanks!

File: pkg/tlshelpers/tlshelpers.go [hunk 1]
```diff
@@ -184,7 +182,8 @@ func CreatePeerCertificate(node *corev1.Node,
 	secretGetter corev1client.SecretsGetter,
 	recorder events.Recorder) (*certrotation.RotatedSelfSignedCertKeySecret, error) {
 	return createCertForNode(
-		fmt.Sprintf("Peer Cert for node %s", node.Name),
+		fmt.Sprintf("Peer (client and server) certificate for node %s, generated by cluster-etcd-operator for etcd. "+
+			"This certificate is valid for %d days and starts refreshing after %d days.", node.Name, durationDays(EtcdCertValidity), durationDays(EtcdCertValidityRefresh)),
```
Comment (hasbro17) [inline pkg/tlshelpers/tlshelpers.go:186]: Do you reckon we should mention the hostnames that our certs are configured to serve? E.g the hardcoded ones plus whatever the node IPs end up being:
https://github.com/openshift/cluster-etcd-operator/blob/767256e7feed840ccedc80d5419e31f3941ce806/pkg/tlshelpers/tlshelpers.go#L56-L67


Only reason I mention it now is to not get nagged at later once we actually update the descriptions doc:
https://github.com/openshift/origin/blob/master/tls/descriptions/descriptions.md#how-to-meet-the-requirement

> Which connections a CA bundle can be used to verify.
What kind of certificates a signer will sign for.
Which names and IPs a serving certificate terminates.
Which subject (user and group) a client certificate is created for.
Which binary and flags is this certificate wired to.

Comment (hasbro17) [reply pkg/tlshelpers/tlshelpers.go:186]: This is the only other example I can find that's mentioning hostnames though:
https://github.com/openshift/origin/blob/master/tls/autoregenerate-after-expiry/autoregenerate-after-expiry.md#certificates-85

Comment (tjungblu) [reply pkg/tlshelpers/tlshelpers.go:186]: @deads2k / @vrutkovs shall we list all those host names / IPs in the annotations?

Comment (vrutkovs) [reply pkg/tlshelpers/tlshelpers.go:186]: No, please don't. These will be censored when collected for TLS registry to keep it consistent - and I don't think its useful for the users

**Comment by tjungblu** [comment]
I think the scaling we're going to override, it's likely not failing on our annotations here :) 

/override ci/prow/e2e-aws-ovn-etcd-scaling

**Pull request merged**